PolyEngine/Engine/Res/debugVertSh.shader

```diff
@@ -0,0 +1,17 @@
+#version 330 core
+layout (location = 0) in vec3 position;
+layout (location = 2) in vec3 normal;
+
+out VS_OUT {
+    vec3 normal;
+} vs_out;
+
+uniform mat4 projection;
+uniform mat4 viewModel;
+
+void main()
+{
+	gl_Position = projection * viewModel * vec4(position, 1.0f); 
```

Comment:
Move `projection * viewModel` to host and pass mvp as single uniform.

Also i suggest using "u_" for uniforms, "a_" for attributes anf "v_" for varyings (in/out)